src/Controller/DefaultRouteController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class DefaultRouteController extends AbstractCookieControllerWithFixedLifetime {
```

Comment:
this controller does not have to inherit from AbstractCookieControllerWithFixedLifetime (doesn't need to inherit from anything, to be precise), we don't need any of the cookie features of that parent class here